src/components/AppBar/AppBar.jsx

```diff
@@ -43,14 +44,18 @@ const styles = (theme) => ({
   },
 });
 
-const AppBar = ({
-  isLoggedIn, selected, classes, routes,
-}) => {
+// eslint-disable-next-line
+const AppBar = ({ isLoggedIn, selected, classes, routes }) => {
   const [showDrawer, setShowDrawer] = useState(false);
   const pageTitle = (routes.find((item) => item.key === selected) || {}).label;
   const history = useHistory();
 
-  const showDrawerItems = () => (
+  const goBack = () => {
+    history.goBack();
+  };
+
+  const showDrawerItems = () =>
+    // eslint-disable-next-line
```

Comment:
Same as below. Do we even need a disable here thought?

Reply:
Removed!